client/src/firebase/FirestoreDocumentReader.js

```diff
@@ -0,0 +1,44 @@
+import { app, database } from './firebase';
+
+/* 
+ * Description: FirestoreDocumentReader reads a firestore collection and converts them
+ *              into custom ad objects.
+ * Author: Kira Toal
+ * Date: 2020/07/14
+ */
+ 
+const COLLECTION = "testing";
+let ads = []; 
+
+class Ad {
+  constructor(id, impressionsMin) {
+    this.id = id;
+    this.impressionsMin = impressionsMin;
+  }
+  toString() {
+    return this.id + ', ' + this.impressionsMin;
+  }
+}
+
+// Firestore data converter converts snapshots to custom objects.
+const adConverter = {
+  toFirestore: function(ad) {
```

Comment:
have a JSDoc explaining `ad` object maybe?

Reply:
+1. Would recommend a JSDoc about the `Ad` class.